docs/guide/velite-schemas.md

```diff
@@ -396,6 +396,38 @@ path: s.path()
 // => flattened path, e.g. 'posts/2021-01-01-hello-world'
 ```
 
+### Parameters
+
+#### **options**: flattening options
+
+- type: `PathOptions`
+
+##### **options.removeIndex**:
+
+Removes index from path for subfolders
+
+- type: `boolean`
+- default: `false`
```

Comment:
I hope the default value is true, which is upward compatible

Reply:
Fixed, it was a mistake. I didn't consider the 'inverted name'